data/en/ships.json

```diff
@@ -292,70 +292,65 @@
 	"Saratoga":"Saratoga",
 	"Commandant Teste":"Commandant Teste",
 
-	"防空棲姫":"Air Defense Princess",
-	"空母棲鬼":"Aircraft Carrier Demon",
-	"空母棲姫":"Aircraft Carrier Princess",
-	"空母水鬼":"Aircraft Carrier Water Demon",
+	"イ":"I",
+	"ロ":"Ro",
+	"ハ":"Ha",
+	"ニ":"Ni",
+	"ホ":"Ho",
+	"ヘ":"He",
+	"ト":"To",
+	"チ":"Chi",
+	"リ":"Ri",
+	"ル":"Ru",
+	"空母ヲ":"Standard Carrier Wo",
+	"ヌ":"Nu",
+	"ワ":"Wa",
+	"カ":"Ka",
+	"ヨ":"Yo",
+	"タ":"Ta",
+	"レ":"Re",
+	"ソ":"So",
+	"ツ":"Tsu",
+	"ネ":"Ne",
+
+	"ナ":"Na",
+	"ラ":"Ra",
+	"ム":"Mu",
+	"ウ":"U",
+	"ヰ":"Wi",
+
+	"空母":"Aircraft Carrier",
+	"戦艦":"Battleship",
```

Comment:
The problem for line 323-327 is that while these are in prefixes, if they are taken away from the ships.json, there will be no root word for BB Hime, CA Hime, CL Hime, etc.